libs/assets/src/lib/asset-details-table.tsx

```diff
@@ -0,0 +1,251 @@
+    tooltip: t('Number of decimal / precision handled by this asset'),
+    value: (asset) => asset.decimals.toString(),
+  },
+  {
+    key: AssetDetail.QUANTUM,
+    label: t('Quantum'),
+    tooltip: t('The minimum economically meaningful amount in the asset'),
+    value: (asset) => asset.quantum,
+  },
+  {
+    key: AssetDetail.STATUS,
+    label: t('Status'),
+    tooltip: t('The status of the asset in the Vega network'),
+    value: (asset) => AssetStatusMapping[asset.status].value,
+    valueTooltip: (asset) => AssetStatusMapping[asset.status].tooltip,
+  },
+  {
+    key: AssetDetail.CONTRACT_ADDRESS,
+    label: t('Contract address'),
+    tooltip: t(
+      'The address of the contract for the token, on the ethereum network'
+    ),
+    value: (asset) => (asset.source as Schema.ERC20).contractAddress,
+  },
+  {
+    key: AssetDetail.WITHDRAWAL_THRESHOLD,
+    label: t('Withdrawal threshold'),
+    tooltip: t(
+      'The maximum allowed per withdraw note: this is a temporary measure for restricted mainnet'
+    ),
+    value: (asset) =>
+      num(asset, (asset.source as Schema.ERC20).withdrawThreshold),
+  },
+  {
+    key: AssetDetail.LIFETIME_LIMIT,
+    label: t('Lifetime limit'),
+    tooltip: t(
+      'The lifetime limits deposit per address note: this is a temporary measure for restricted mainnet'
+    ),
+    value: (asset) => num(asset, (asset.source as Schema.ERC20).lifetimeLimit),
+  },
+  {
+    key: AssetDetail.MAX_FAUCET_AMOUNT_MINT,
+    label: t('Max faucet amount'),
+    tooltip: t(
+      'Maximum amount that can be requested by a party through the built-in asset faucet at a time'
+    ),
+    value: (asset) =>
+      num(asset, (asset.source as Schema.BuiltinAsset).maxFaucetAmountMint),
+  },
+  {
+    key: AssetDetail.INFRASTRUCTURE_FEE_ACCOUNT_BALANCE,
+    label: t('Infrastructure fee account balance'),
+    tooltip: t('The infrastructure fee account for this asset'),
+    value: (asset) => num(asset, asset.infrastructureFeeAccount.balance),
+  },
+  {
+    key: AssetDetail.GLOBAL_REWARD_POOL_ACCOUNT_BALANCE,
+    label: t('Global reward pool account balance'),
+    tooltip: t('The global reward pool account for this asset'),
+    value: (asset) => num(asset, asset.globalRewardPoolAccount?.balance),
+  },
+  {
+    key: AssetDetail.TAKER_FEE_REWARD_ACCOUNT_BALANCE,
+    label: t('Taker fee reward account balance'),
+    tooltip: t('The taker fee reward account for this asset'),
+    value: (asset) => num(asset, asset.takerFeeRewardAccount?.balance),
+  },
+  {
+    key: AssetDetail.MAKER_FEE_REWARD_ACCOUNT_BALANCE,
+    label: t('Maker fee reward account balance'),
+    tooltip: t('The maker fee reward account for this asset'),
+    value: (asset) => num(asset, asset.makerFeeRewardAccount?.balance),
+  },
+  {
+    key: AssetDetail.LP_FEE_REWARD_ACCOUNT_BALANCE,
+    label: t('Liquidity provision fee reward account balance'),
+    tooltip: t('The liquidity provision reward account for this asset'),
```

Comment:
This tooltip (and some others in this section) can work a little harder. Maybe something like "The rewards acquired based on the liquidity provision fees you've received".